tests/framework/web/UrlRuleTest.php

```diff
@@ -51,8 +51,8 @@ public function testParseRequest()
                 $request->pathInfo = $test[0];
                 $expected = $test[1];
                 $result = $rule->parseRequest($manager, $request);
-                if ($expected === false) {
-                    $this->assertFalse($result, "Test#$i-$j: $name");
+                if (empty($expected)) {
```

Comment:
Ah, you're right, this should be done only for `createUrl()` tests. I will revert this.